src/pages/OfferNew/Checkout/hooks.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  CompleteQuote,
+  SignState,
+  useMemberQuery,
+  useRedeemedCampaignsQuery,
+} from 'data/graphql'
+import * as React from 'react'
+import { InsuranceType } from 'utils/insuranceDomainUtils'
+import { adtraction, trackStudentkortet } from 'utils/tracking'
+
+export enum VisibilityState {
+  CLOSED,
+  CLOSING,
+  OPENING,
+  OPEN,
+}
+
+export const useScrollLock = (
```

Comment:
This scrollLock doesn't  work for me, `VisibilityState` never seems to be set to `OPEN ` in the hook.

Reply:
Wow, really nice catch! Fixed now

Reply:
Awesome! 🥳